src/lib/pages/backoffice/Patron/PatronDetails/ItemsSearch/ItemsResultsList/ItemsResultsList.js

```diff
@@ -52,56 +52,74 @@ export default class ItemsResultsList extends Component {
       circulationStatus
     ) && invenioConfig.ITEMS.canCirculateStatuses.includes(row.metadata.status)
       ? buttonCheckout
-      : this.renderForceCheckoutModal(row);
+      : this.renderForceCheckoutModal(
+          row,
+          circulationStatus === 'ITEM_ON_LOAN'
+        );
   };
 
-  renderForceCheckoutModal = item => {
+  renderForceCheckoutModal = (item, isItemOnLoan) => {
     const { isModalOpen } = this.state;
     const { patronPid } = this.props;
     const itemPid = {
       type: recordToPidType(item),
       value: item.metadata.pid,
     };
+
     return (
-      <Modal
-        trigger={
-          <Button color="yellow" onClick={this.toggleModal}>
-            Force Checkout
-          </Button>
-        }
-        open={isModalOpen}
-        onClose={this.toggleModal}
-        closeIcon
-      >
-        <Header
-          icon="exclamation triangle"
-          content="Force Checkout of Missing Item"
-        />
-        <Modal.Content>
-          <p>
-            You are creating a loan for a missing item, would you like to
-            continue?
-          </p>
-        </Modal.Content>
-        <Modal.Actions>
-          <Button color="red" onClick={this.toggleModal}>
-            <Icon name="remove" /> No
-          </Button>
-          <Button
-            color="green"
-            onClick={() =>
-              this.onClickCheckoutHandler(
-                item.metadata.document_pid,
-                itemPid,
-                patronPid,
-                true
-              )
+      <>
+        <Modal
+          trigger={
+            <Button
+              color="yellow"
+              onClick={this.toggleModal}
+              disabled={isItemOnLoan}
+            >
+              Force Checkout
+            </Button>
+          }
+          open={isModalOpen}
+          onClose={this.toggleModal}
+          closeIcon
+        >
+          <Header
+            icon="exclamation triangle"
+            content="Force Checkout of Missing Item"
```

Comment:
There might be other reasons that the item cannot circulate other than `missing`. Maybe `Force Checkout` is enough.

We should align how we type Check-in and check-out 

- check-in
- checkin
- Checkin
- CheckIn
- Check-In
..

For variable names I have used `CheckOut` everywhere
For display (header, sidebar etc) `Check-out`

what do you suggest @KonstantinaStoikou ? and maybe the naming police @kprzerwa has an opinion.

Reply:
I found those things:
- https://grammarist.com/spelling/checkout-check-out/
- https://grammarist.com/spelling/check-in/
that explain the correct naming

Reply:
👍  to harmonize check-in/out everywhere with the correct spelling!